src/Indicator.vala

```diff
@@ -171,17 +172,24 @@ public class Notifications.Indicator : Wingpanel.Indicator {
         }
 
         set_display_icon_name ();
+        update_tooltip ();
     }
 
     private void update_clear_all_sensitivity () {
         clear_all_btn.sensitive = nlist.app_entries.size > 0;
+
+        /* Since update_clear_all_sensitivity() is called on
+        each popover remove, we can update our tooltip on
+        each popover remove here. */
```

Comment:
This should be "notification bubble" instead of "popover" since popover is a specific thing in Gtk